src/FSharp.Core/Query.fs

```diff
@@ -722,21 +722,21 @@ module Query =
             (if isIQ then CQ else CE) ([srcItemTy], [src])
         Make, Call
 
-    let MakeFirst, CallFirst = MakeOrCallSimpleOp (methodhandleof (fun x -> System.Linq.Queryable.First x)) (methodhandleof (fun x -> Enumerable.First x))
```

Comment:
@dotnet/fsharp-team-msft I wonder if it may be any different when passing method group vs FSharpFunc for queries... Wonder if ORMs care.

Reply:
I don't think your ping worked... I don't have answer for this one. As ORM maintainer myself (SQLProvider), I've previously had to just adapt to whatever changes in LINQ-trees that have came. But also the C# LINQ expression-trees are already working without FSharpFunc, and F# side often deals with active-pattern similar to (|OptionalCoerce|) in this code-base.

Reply:
Ping worke, just many people from team are out, and don't react right away

Reply:
Tested a LINQ SQLProvider project with this branch dll, worked without any issues.

Reply:
@vzarytovskii I think this is the only thing pending the merge, so you can merge when happy :-)